### PR TITLE
Add sdw-dom0-config 0.11.1-rc1

### PR DESCRIPTION
**Pull request by rocodes**
###
Name of package: securedrop-workstation-dom0-config-0.11.1-rc1

Refs: https://github.com/freedomofpress/securedrop-workstation/issues/1057

### Test plan

- [x] Tag in securedrop-workstation repository is correct: https://github.com/freedomofpress/securedrop-workstation/releases/tag/0.11.1-rc1
- [x] Build logs are included: https://github.com/freedomofpress/build-logs/commit/da4f266
- [ ] Build is reproducible from 0.11.1-rc1 tag

### CODE REVIEW

**Comment by zenmonkeykstop** [comment]
Which tag should we be using to check reproducibility?

**Comment by rocodes** [comment]
> Which tag should we be using to check reproducibility?

Sorry - the 0.11.1-rc1 tag (I have amended the PR body) :  https://github.com/freedomofpress/securedrop-workstation/releases/tag/0.11.1-rc1

**Review by cfm — APPROVED** [review]
- [x] Tag in securedrop-workstation repository is correct: https://github.com/freedomofpress/securedrop-workstation/releases/tag/0.11.1-rc1
- [x] Build logs are included: https://github.com/freedomofpress/build-logs/commit/da4f266
- [ ] Build is reproducible from 0.11.1-rc1 tag 

No, per diffoscope:

```
├── content
│ ├── file list
│ │ @@ -77,13 +77,13 @@
│ │  -rw-r--r--   1        0        0      666 2024-06-02 00:00:00.000000 ./usr/lib/python3.8/site-packages/securedrop_workstation_dom0_config-0.11.1rc1.dist-info/RECORD
│ │  -rw-r--r--   1        0        0       92 2024-06-02 00:00:00.000000 ./usr/lib/python3.8/site-packages/securedrop_workstation_dom0_config-0.11.1rc1.dist-info/WHEEL
│ │  -rw-r--r--   1        0        0        1 2024-06-02 00:00:00.000000 ./usr/lib/python3.8/site-packages/securedrop_workstation_dom0_config-0.11.1rc1.dist-info/top_level.txt
│ │  -rw-r--r--   1        0        0     1060 2024-06-02 00:00:00.000000 ./usr/lib/systemd/user/securedrop-user-dom0-rpm-key-reimport.service
│ │  drwxr-xr-x   1        0        0        0 2024-06-02 00:00:00.000000 ./usr/share/doc/securedrop-workstation-dom0-config
│ │  -rw-r--r--   1        0        0    16107 2024-06-02 00:00:00.000000 ./usr/share/doc/securedrop-workstation-dom0-config/README.md
│ │  drwxr-xr-x   1        0        0        0 2024-06-02 00:00:00.000000 ./usr/share/licenses/securedrop-workstation-dom0-config
│ │ --rw-r--r--   1        0        0    34520 2022-08-23 00:22:50.000000 ./usr/share/licenses/securedrop-workstation-dom0-config/LICENSE
│ │ +-rw-r--r--   1        0        0    34520 2024-04-24 22:02:05.000000 ./usr/share/licenses/securedrop-workstation-dom0-config/LICENSE
│ │  -rw-r--r--   1        0        0      320 2024-06-02 00:00:00.000000 ./usr/share/securedrop-workstation-dom0-config/config.json.example
│ │  -rwxr-xr-x   1        0        0      946 2024-06-02 00:00:00.000000 ./usr/share/securedrop-workstation-dom0-config/scripts/clean-salt
│ │  -rwxr-xr-x   1        0        0     1993 2024-06-02 00:00:00.000000 ./usr/share/securedrop-workstation-dom0-config/scripts/destroy-vm
│ │  -rwxr-xr-x   1        0        0     1815 2024-06-02 00:00:00.000000 ./usr/share/securedrop-workstation-dom0-config/scripts/provision-all
│ │  -rwxr-xr-x   1        0        0     5645 2024-06-02 00:00:00.000000 ./usr/share/securedrop-workstation-dom0-config/scripts/validate_config.py
```

This is just like <https://github.com/freedomofpress/securedrop-yum-test/pull/58#issuecomment-2194937742>, except that @legoktm's hypothesis there concerns <https://github.com/freedomofpress/securedrop-workstation/commit/15d61fe52e3aa731ec0032294fa3e7f61b92d89a>, which isn't part of `release/0.11.1`.  But to unspool it a bit: that commit *removed* special mtime-clamping for the `LICENSE` file, and the removed behavior may not have been correct or tested fully anyway.

Since this is the only diff according to diffoscope, I'll approve tentatively based on <https://github.com/freedomofpress/securedrop-yum-test/pull/58#issuecomment-2195148156> but defer to you as release manager, @rocodes, for merge.

**Comment by rocodes** [comment]
Thanks @cfm. I don't love it, but for now I'll settle for local reproducibility (`make reprotest`).

```
[snip]
Build complete! RPMs and their checksums are:

e78ecbf77b785950c0d6411b52ca628e286bab30709a2759dfa8298a9bd95c92  rpm-build/RPMS/noarch/securedrop-workstation-dom0-config-0.11.1rc1-1.fc32.noarch.rpm
=======================
Reproduction successful
=======================
No differences in ./rpm-build/RPMS/noarch/*.rpm
e78ecbf77b785950c0d6411b52ca628e286bab30709a2759dfa8298a9bd95c92  ./rpm-build/RPMS/noarch/securedrop-workstation-dom0-config-0.11.1rc1-1.fc32.noarch.rpm

```

**Comment by legoktm** [comment]
I've updated https://github.com/freedomofpress/securedrop-workstation/pull/1114 so it can also be used for the 0.11.1 branch.

**Comment by rocodes** [comment]
Going to hold off on merge until an update in https://github.com/freedomofpress/securedrop-workstation/pull/1114 that will allow rpms to be reproducible. So we may go to rc2 and not even deploy this one. @cfm

**Comment by rocodes** [comment]
Closing - rc2 is on the way and reproducibility issues have been addressed

**Pull request closed without merging**
